ntc_templates/templates/huawei_smartax_display_ont_info_0_1_2.textfsm

```diff
@@ -1,19 +1,21 @@
 Value Required FSP (\w+\/\s\w+\/\w+)
 Value Key ONT_ID (\d+)
-Value SERIAL_NUMBER (\w+)
-Value CONTROL_FLAG (\w+)
-Value RUN_STATE (\w*)
-Value CONFIG_STATE (\w*)
-Value MATCH_STATE (\w*)
+Value SERIAL_NUMBER ([A-Z0-9]{16})
+Value CONTROL_FLAG (active|configuring|)
```

Comment:
```suggestion
Value CONTROL_FLAG (active|configuring|)
```
did you intend to have the trailing `|`?

Reply:
Sorry I forgot the deactive flag also 😅. I have updated the template